accelerator/cli_info.h

```diff
@@ -86,6 +88,8 @@ static struct ta_cli_argument_s {
     {"health_track_period", no_argument, NULL, HEALTH_TRACK_PERIOD,
      "The period for checking IRI host connection status"},
     {"no-gtta", no_argument, NULL, NO_GTTA, "Disable getTransactionToConfirm (gTTA) when sending transacation"},
+    {"buffer_list", required_argument, NULL, BUFFER_LIST, "Set the name of buffer_list_name"},
```

Comment:
How about "set the name of buffer_list" or "set the value of buffer_list_name"?

Reply:
The option is hard to understand.